### PR TITLE
Refactor nginx container build

### PR DESCRIPTION
**Pull request by dchiller**
I just went ahead and did this because it was pretty simple and helps with increasing the speed of container rebuilds on modifications to javascript. I don't want to close the nginx container build issue because there are still some questions (eg. what's up with those errant node_modules), but this does increase speed.

### CODE REVIEW

File: nginx/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 WORKDIR /code/frontend
+RUN npm install
+COPY ./public/node/ /code/
 RUN wget http://www.fawe.de/volpiano/volpiano51_web.zip
 RUN unzip volpiano51_web.zip
 RUN rm volpiano51_web.zip
 RUN mv volpiano51_web/volpiano.woff ../static/fonts/volpiano.woff
-RUN npm install
-RUN gulp build --release
+RUN gulp build
```
Comment (napulen) [inline nginx/Dockerfile:12]: I'm curious about the removal of the `--release` flag in `gulp build`. I thought this made the difference between minified and not-minified `js`. We want minified by default, except when debugging, right?

Comment (napulen) [reply nginx/Dockerfile:12]: Also, can we remove the `npm install`?

Comment (dchiller) [reply nginx/Dockerfile:12]: > Also, can we remove the `npm install`?

No, but I moved it to a different stage in the process...see new line 6.

Comment (dchiller) [reply nginx/Dockerfile:12]: > I'm curious about the removal of the `--release` flag in `gulp build`. I thought this made the difference between minified and not-minified `js`. We want minified by default, except when debugging, right?

Ah, yes...that's an error...I was debugging when I committed and forgot to change it back. Will do now.

**Review by napulen — APPROVED** [review]

**Pull request merged**